src/components/User/UserItem.vue

```diff
@@ -10,15 +10,15 @@ import { getCurrentUser } from '@nextcloud/auth'
 import { t } from '@nextcloud/l10n'
 
 import NcAvatar from '@nextcloud/vue/components/NcAvatar'
-import AdminIcon from 'vue-material-design-icons/ShieldCrown.vue'
+import AdminIcon from 'vue-material-design-icons/ShieldCrownOutline.vue'
 import LinkIcon from 'vue-material-design-icons/LinkVariant.vue'
```

Comment:
currently unifying the move from `LinkVariant` to  `Link`
Not mandatory in any way of course if you want to stick to the classic "chain" icon than the (more) Material one